docs/api/upgrade-assistant/batch_reindexing.asciidoc

```diff
@@ -0,0 +1,81 @@
+[[batch-start-resume-reindex]]
+=== Batch start or resume reindex API
+++++
+<titleabbrev>Batch start or resume reindex</titleabbrev>
+++++
+
+Enables starting or resuming multiple reindexing tasks in one request. Additionally, reindexing tasks started or resumed
+via the batch endpoint will be placed on a queue and executed one-by-one which ensures that minimal cluster resources
+are consumed over time.
+
+experimental["The underlying Upgrade Assistant concepts are stable, but the APIs for managing Upgrade Assistant are experimental."]
+
+[[batch-start-resume-reindex-request]]
+==== Request
+
+`POST /api/upgrade_assistant/reindex/batch`
+
+[[batch-start-resume-reindex-request-body]]
+==== Request body
+
+`indexNames`::
+  (Required, array) The list of index names to be reindexed.
+
+[[batch-start-resume-reindex-codes]]
+==== Response code
+
+`200`::
+  Indicates a successful call.
+
+[[batch-start-resume-example]]
+==== Example
+
+[source,js]
+--------------------------------------------------
+POST /api/upgrade_assistant/reindex/batch
+{
+  "indexNames": [ <1>
+    "index1",
+    "index2"
+  ]
+}
+--------------------------------------------------
+<1> The order in which the indices are provided here determines the order in which the reindex tasks will be executed.
+
+The API returns the following which is similar to the <<start-resume-reindex, start or resume endpoint>>:
```

Comment:
There is a missing comma between `following` and `which`.

Or I would switch this around. For example:

`Similar to the <<start-resume-reindex, start or resume endpoint>>, the API returns the following:`